pylintrc

```diff
@@ -19,35 +19,36 @@ score=yes
 
 [MESSAGES CONTROL]
 disable=
+ abstract-method,
+ arguments-differ,
+ attribute-defined-outside-init,
+ broad-except,
+ broad-exception-raised,
+ consider-using-f-string,
+ duplicate-code,
  fixme,
+ inconsistent-return-statements,
  invalid-name,
- no-value-for-parameter,
- pointless-statement,
- missing-module-docstring,
  missing-class-docstring,
  missing-function-docstring,
+ missing-module-docstring,
+ no-member,
+ no-self-use,
+ no-value-for-parameter,
+ pointless-statement,
+ protected-access,
+ super-init-not-called,
+ too-few-public-methods,
+ too-many-ancestors,
  too-many-arguments,
- too-many-locals,
  too-many-branches,
  too-many-instance-attributes,
- too-many-ancestors,
- too-many-public-methods,
  too-many-lines,
+ too-many-locals,
  too-many-nested-blocks,
+ too-many-public-methods,
  too-many-statements,
- protected-access,
```

Comment:
this reordering makes it hard to see what has been enabled and disabled. is there anyway we could maintain the original order so that diff only shows what has changed?

Reply:
Ok, will do that